packages/client/hmi-client/src/components/workflow/ops/compare-datasets/compare-datasets-utils.ts

```diff
@@ -0,0 +1,31 @@
+import { WorkflowNode } from '@/types/workflow';
+import { computed, Ref } from 'vue';
+import { DataArray } from '@/services/models/simulation-service';
+import _ from 'lodash';
+import { CompareDatasetsState } from './compare-datasets-operation';
+
+export function usePreparedChartInputs(
+	props: {
```

Comment:
`usePreparedChartInputs` seems to be unrelated to the scenario template change here?

Reply:
Oops, that snuck in there will remove this file